Omnibus.md

```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
+spell which removes the PC's [soul. Soul tokens]{.new} must have their
+player's name and the PC's name written legibly on it. A [soul token]{.new}
+does not exist physically in the IC world therefore it cannot be searched. A
+[soul token]{.new} is a non-magical and non-stealable marshaling tool.
+
+### [Destroying a Body]{.new} ### {#destroying-a-body}
+
+In order [for a player to separate a soul from a body, they must destroy
+it by]{.new} striking 200 blows with a weapon beside the body of the character 
+being [destroyed]{.new}. More than one person may [destroy]{.new} a body at 
+a time. More than one weapon may be used to [destroy]{.new} a body as well. This 
+effectively divides the number of blows to be struck between the number of
+participants and number of weapons used. Some monsters and characters under
+certain spells may require more blows to completely [destroy]{.new}. If you
+strike 200 blows and the victim says "The job is not yet done," then the [body
+is not yet destroyed]{.new}. Some spells, such as
+[Strange Brew: Potion of Acid](#potion-of-acid) and
+[Assassin's Blade](#assassins-blade), may accelerate the act of [destroying a
+body]{.new}.
+
+After the [body is destroyed]{.new}, the character whose body has been
+destroyed is [rendered soulless]{.new}.  The player of the
+[soulless]{.new} character must present their character's [soul]{.new}
+token to the character that just [destroyed their body]{.new}. It is
+then that player's responsibility to present the [soul]{.new} token to an
+EH or an appointed marshal. The EH must be informed immediately of the
+[body's destruction]{.new}, thus allowing the EH time to prepare for those
+who might wish to return the [soulless]{.new} character to life.
+
+Characters that are pretending to be dead should interpret any
+[body destroying]{.new} blows as blows to the closest kill location. So, if
+your PC is pretending to be dead and someone starts to [destroy their
+body]{.new}, the first blows that are struck on the ground next to you should be
+played as if they were striking your PC on the nearest kill location,
+destroying any undamaged armor, and then killing them.
+
+If your PC is killed and [their body is destroyed]{.new}, they will not
+remember anything about how they died should they manage to be raised.
+
+Should your PC be dead at the end of an event without being raised,
+even if the character['s body was not actively destroyed]{.new}, the PC will be
+considered [soulless]{.new}.
+
+### Fixing a [Soulless]{.new} Character ### {#fixing-a-soulless-character}
+
+To restore someone to life after [they have lost their soul]{.new},
+characters must first have the body of the person needing to be raised. Then
+characters must either cast a [Call the Soul](#call-the-soul), an
+[Intervention](#intervention) spell, or administer a
+[Potion of Soul Snare](#potion-of-soul-snare)
+(see [Strange Brew](#strange-brew)) to summon and reattach the soul. If
```

Comment:
`(see [Strange Brew](#strange-brew)) to summon and reattach the [soul]{.new}. If`